docker/docker-compose.mysql-nossl-server-ssl-connector_transparent.yml

```diff
@@ -0,0 +1,169 @@
+        volumes:
+            - ./.acrakeys:/keys
+        command: >-
+            --client_id=${ACRA_CLIENT_ID:-testclientid}
+            --generate_acrawriter_keys
+            --keys_output_dir=/keys/acra-server
+            --keys_public_output_dir=/keys/acra-server
+
+    # MySQL container
+    mysql:
+        image: mariadb:10.3
+        # INSECURE!!! You MUST define your own DB name and credentials
+        environment:
+            MYSQL_ONETIME_PASSWORD: ${MYSQL_ONETIME_PASSWORD:-urLK7YJPndDWsnJC}
+            MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD:-hy53uBLmBuihUK29}
+            MYSQL_DATABASE: ${MYSQL_DATABASE:-test}
+            MYSQL_USER: ${MYSQL_USER:-test}
+            MYSQL_PASSWORD: ${MYSQL_PASSWORD:-test}
+        # Open the port outside for writer
+        ports:
+            - "3306:3306"
+        # We use internal 'server-mysql' network for AcraServer and
+        # DB interconnection and external network 'world' for port exposing
+        networks:
+            - server-mysql
+            - world
+
+    acra-server:
+        build:
+            context: ../
+            dockerfile: docker/acra-server_transparent.dockerfile
+        # Restart server after correct termination, for example after the config
+        # was changed through the API
+        restart: always
+        depends_on:
+            - acra-keymaker_server
+            - acra-keymaker_connector
+            - acra-keymaker_writer
+            - mysql
+        environment:
+            ACRA_MASTER_KEY: ${ACRA_SERVER_MASTER_KEY:-N0MyZEVCRGY1cHA4ODRSVHp3c3BNeGtDekxaV3BhTHI=}
+        # We use internal networks:
+        # - 'server-mysql' - for AcraServer and DB interconnection
+        # - 'connector-server' - for AcraServer and AcraConnector interconnection
+        networks:
+            - connector-server
+            - server-mysql
+        volumes:
+            # Mount the directory with only the keys for this service. Must be
+            # rewriteable in case of using API, otherwise should be read-only.
+            - ./.acrakeys/acra-server:/keys
+            # Directory with configuration, rewriteable
+            - ./.acraconfigs/acra-server:/config
+            # Mount directories with SSL certificates
+            - ./ssl/ca:/ssl.ca:ro
+            - ./ssl/acra-server:/ssl.server:ro
+        command: >-
+            --mysql_enable
+            --db_host=mysql
+            --db_port=3306
+            --keys_dir=/keys
+            --client_id=${ACRA_CLIENT_ID:-testclientid}
+            --auth_keys=/keys/httpauth.accounts
+            --http_api_enable
+            --incoming_connection_api_string=tcp://0.0.0.0:9090
+            --config_file=/config/acra-server.yaml
+            --acraconnector_tls_transport_enable
+            --tls_ca=/ssl.ca/example.cossacklabs.com.crt
+            --tls_cert=/ssl.server/acra-server.crt
+            --tls_key=/ssl.server/acra-server.key
+            --acraconnector_transport_encryption_disable
```

Comment:
this option is redundant and will not used. It works in next [manner](https://github.com/cossacklabs/acra/blob/master/cmd/acra-server/acra-server.go#L234):
* use tls if `acraconnector_tls_transport_enable` used
* don't encrypt anything if `acraconnector_transport_encryption_disable`
* use secure session otherwise

same for connector

Reply:
I see. Going to change this in next commits